src/BlazorUI/Bit.BlazorUI.Tests/SnackBar/BitSnackBarWithBodyTemplate.razor

```diff
@@ -0,0 +1,6 @@
+<BitSnackBar>
+    <BodyTemplate Context="body">
+        <p>@body</p>
+        <span>second element</span>
```

Comment:
either remove this line or accept it as a parameter, since it is part of the expected value in the test.

Reply:
The idea behind this is when I tried to assert without removing unwanted children I have to select second child element to prevent Title header and progressbar. this will pass but when I was thinking what if user have two element which not wrapped to a element. So my test failed after adding this line and I've had to change the test to what you mentioned as a problem to removing child like this

Reply:
The idea behind this is when I tried to assert without removing unwanted children I have to select second child element to prevent Title header and progressbar. this will pass but when I was thinking what if user have two element which not wrapped to a element. So my test failed after adding this line and I've had to change the test to what you mentioned as a problem to removing child like this

Reply:
https://bunit.dev/docs/verification/verify-component-state.html
https://bunit.dev/docs/verification/semantic-html-comparison.html